diff --git a/dist/popup.js b/dist/popup.js
index 4e43272dcf60a8c5580c85ba71efbf3e8b2765fc..7eb9ebc735b76cf8267f469f7d8b1d66aee408fb 100644
--- a/dist/popup.js
+++ b/dist/popup.js
@@ -37,6 +37,7 @@ dialog::backdrop{
   inset: 0;
   width: 100%;
   height: 100%;
+  cursor: default;
 }
 .container{
   display: block;
diff --git a/dist/ripple.js b/dist/ripple.js
index fa7d861ed4a578d23cb33b69d8d1d9f6f545c465..477db8912c85947887f24be876f870fcc443868a 100644
--- a/dist/ripple.js
+++ b/dist/ripple.js
@@ -40,7 +40,7 @@ const style = /*css*/ `
   opacity: 0;
   border-radius: 50%;
   background: currentColor;
-  filter: blur(12px) opacity(var(--ripple-opacity, .18));
+  filter: opacity(var(--ripple-opacity, .18));
 }
 `;
 const template = /*html*/ `
